src/main/java/com/google/sps/servlets/utils/YouTubeCommentRetriever.java

```diff
@@ -37,7 +37,7 @@
 
   private final String APPLICATION_NAME = "SAY";
   // TODO: have dev key come from centralized location, rather than being hard-coded.
-  private final String DEVELOPER_KEY = "AIzaSyDYfjWcy1hEe0V7AyaYzgIQm_rT-9XbiGs";
+  private final String DEVELOPER_KEY = KeyRetriever.getApiKey();
 
   private YouTube youtubeService = null;
 
```

Comment:
general comments: it probably needs to be changed in js as well?

Reply:
Js? What do you mean? the API Key is not used in any javascripts is it?

Reply:
It's used in context and search js files